src/selector/scrollTo.js

```diff
@@ -12,6 +12,24 @@ define([
         $target = $target || $('body');
         options = options || {};
 
+        if (options.container) {
```

Comment:
How about:

```
options.container = options.container && $(options.container).get(0);

if ($container.css('position') === 'static') {
    throw options.container + ' needs to have a non-static position property';
}

```

This will ensure that the container object is a plain DOM element, and assign it if exists.